Corona-Warn-App/src/main/res/layout/include_submission_positive_other_warning.xml

```diff
@@ -51,18 +60,39 @@
                 app:layout_constraintStart_toStartOf="@id/guideline_start"
                 app:layout_constraintTop_toBottomOf="@+id/submission_positive_other_warning_headline" />
 
+            <TextView
+                android:id="@+id/submission_country_header_description"
+                style="@style/headline6"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="24dp"
+                android:text="@{@string/submission_interoperability_list_title}"
+                app:layout_constraintEnd_toEndOf="@id/guideline_end"
+                app:layout_constraintStart_toStartOf="@id/guideline_start"
+                app:layout_constraintTop_toBottomOf="@+id/submission_positive_other_warning_text" />
+
+            <de.rki.coronawarnapp.ui.view.CountryList
```

Comment:
Instead of using the TextView above you could use the property "countryListTitle" of the "CountryList" view. Should have the same effect and reduce the xml file here

Reply:
You're right, I'll change it to the countryListTitle property.

Reply:
Wait, this can't be done. I haven't included the whole fragment with the country list, just the list itself, otherwise I would have had to make changes to the country list fragment too.

Reply:
Yeah you're right was my bad. Though you used the include layout 👍